source/usage-examples/findMany.txt

```diff
@@ -0,0 +1,170 @@
+      :tabid: linq-async
+
+      Find Documents Using LINQ
+      ~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      The following example uses LINQ to asynchronously find documents in the
+      ``restaurants`` collection with the ``cuisine`` "Pizza".
+
+      .. literalinclude:: ../includes/code-examples/FindManyAsync.cs
+         :start-after: start-find-linq-async
+         :end-before: end-find-linq-async
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+   .. tab:: LINQ Synchronous
+      :tabid: linq-sync
+
+      Find a Document Using LINQ
+      ~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+      The following example uses LINQ to synchronously find documents in the
+      ``restaurants`` collection with the ``cuisine`` "Pizza".
+
+      .. literalinclude:: ../includes/code-examples/FindManySync.cs
+         :start-after: start-find-linq-sync
+         :end-before: end-find-linq-sync
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+Find All Documents
+~~~~~~~~~~~~~~~~~~
+
+.. tabs::
+
+   .. tab:: Async
+      :tabid: find-all-async
+
+      The following example asynchronously finds all documents in the 
+      ``restaurants`` collection.
+
+      .. literalinclude:: ../includes/code-examples/FindManyAsync.cs
+         :start-after: start-find-all-async
+         :end-before: end-find-all-async
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+   .. tab:: Synchronous
+      :tabid: find-all-sync
+
+      The following example synchronously finds all documents in the 
+      ``restaurants`` collection.
+
+      .. literalinclude:: ../includes/code-examples/FindManySync.cs
+         :start-after: start-find-all-sync
+         :end-before: end-find-all-sync
+         :language:  csharp
+         :copyable:
+         :dedent:
+
+Expected Result
+~~~~~~~~~~~~~~~
+
+Running the :ref:`full example <find-runnable-example>` returns 
+the following results:
+
+.. code-block:: none
+
+    Finding documents with builders...:
+    Number of documents found: 1163
+
+    Finding documents with LINQ...:
+    Number of documents found: 1163
+
+    Finding all documents...:
+    Number of documents found: 25359
+
+.. tip:: Sample Datasets
+   
+   These examples use the :atlas:`sample datasets </sample-data>` provided by Atlas.
+   The number of documents returned may differ depending on the data in your
+   collection.
+
+Additional Information
+----------------------
+
+.. _find-many-runnable-example:
+
+For a fully runnable example of the ``Find`` operations on this page, see the following:
+
+- `Async Find Multiple Example <{+example+}/FindManyAsync.cs>`__
+- `Synchronous Find Multiple Example <{+example+}/FindManySync.cs>`__
+
+.. TODO: Add links to references once pages are done
+
+To learn more about builders, see <builders page>.
+
+To learn more about LINQ, see <LINQ page>.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+`Find() <{+api-root+}/html/Overload_MongoDB_Driver_IMongoCollectionExtensions_Find.htm>`__
```

Comment:
I used bullets here for an unordered list. Preference?

Reply:
Agreed on bullets - changed